File: Library/PaintedSkin/src/main/java/org/alee/component/skin/pack/AssetManagerMapping.java
package org.alee.component.skin.pack;

import android.content.res.AssetManager;

import org.alee.reflex.ReflexClass;
import org.alee.reflex.ReflexMethod;
import org.alee.reflex.annotation.MethodParams;

/**********************************************************
 *
 * @author: MY.Liu
 * @date: 2021/2/13
 * @description: xxxx
 *
 *********************************************************/
final class AssetManagerMapping {
    @MethodParams(String.class)
    public static ReflexMethod<Integer> addAssetPath;

    static {
        ReflexClass.load(AssetManagerMapping.class, AssetManager.class);
    }
}

File: Library/PaintedSkin-Kotlin/src/main/java/android/content/res/AssetManagerMapping.kt
package android.content.res

import org.alee.component.skin.util.INotProguard
import org.alee.reflex.ReflexClass
import org.alee.reflex.ReflexConstructor
import org.alee.reflex.ReflexMethod
import org.alee.reflex.annotation.MethodParams
import org.alee.reflex.annotation.ReflexMeta

/**
 * 摘要
 *
 * <p> 详细描述
 *
 * @author MingYu.Liu
 * created in 2023/8/31
 *
 */
internal class AssetManagerMapping : INotProguard {
    companion object {
        lateinit var constructor: ReflexConstructor<AssetManager>

        @ReflexMeta
        @MethodParams(String::class)
        lateinit var addAssetPath: ReflexMethod<Int>

        init {
            ReflexClass.load(AssetManagerMapping::class.java, AssetManager::class.java)
        }
    }
}
